relayers/erc721_relayer.go

```diff
@@ -129,41 +129,46 @@ func (relayer *ERC721Relayer) Address() (common.Address, error) {
 	return relayer.address, nil
 }
 
-func (relayer *ERC721Relayer) Validate(recipient common.Address, tokenID, sourceID, sourceTokenID, liveUntil *big.Int, metadataURI string, authorizationMessage interface{}) (bool, error) {
+// Validate handles signature validation.
+// Important! All errors going to client response, do not pass signatures and valuable information.
+func (relayer *ERC721Relayer) Validate(recipient common.Address, tokenID, sourceID, sourceTokenID, liveUntil *big.Int, metadataURI string, authorizationMessage interface{}) error {
```

Comment:
Define list of errors and response only specified error to user like:
`var ErrUnauthorizedRequest error = errors.New("unauthorized request")`